### DISCUSSION TITLE
Gatsby Cloud. I keep receiving The Workspace is now in idle mode

### DISCUSSION THREAD

**Discussion by keeko20** [gatsbyjs/gatsby#30775] [Help] [answered]
Email says
"It's been over 14 days since a real-time edit was made in any site in the (username) Workspace on Gatsby Cloud."

I almost reactivating every day, It says 14 days but it's not even 14hrs have passed then it will back to idle mode?

**Comment by LekoArts** [comment, marked as answer]
For Gatsby Cloud support please reach out here: https://www.gatsbyjs.com/support/
Thanks!

**Reply by keeko20** [reply]
Right, found a sales form only before. isn't easy to find that support form. Thanks!
